src/webhooks/webhooks.service.ts

```diff
@@ -63,13 +64,29 @@ export class WebhooksService {
         therapySessionsRemaining: Number(partnerAccessDetails.therapySessionsRemaining) - 1,
         therapySessionsRedeemed: Number(partnerAccessDetails.therapySessionsRedeemed) + 1,
       };
+
+      await updateCrispProfile(
+        {
+          therapy_sessions_remaining: Number(partnerAccessDetails.therapySessionsRemaining) - 1,
+          therapy_sessions_redeemed: Number(partnerAccessDetails.therapySessionsRedeemed) + 1,
+        },
```

Comment:
can we just use

```
await updateCrispProfile(
        partnerAccessUpdateDetails,
       client_email,
};
```
we dont need to repeat the values here

Reply:
i've pushed this change and removed the `Number(` before these values, as they're already numbers?